### PR TITLE
First version of right click menu

### PR DESCRIPTION
**Pull request by michaelw8**
[ch328]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +16,43 @@ Installation
...
+Setup
+------------------------------------------------------------------------------
+The right click menu needs some styling, which must be included manually. Add one of the following imports to your application. If you don't want any custom styling and style the right click menu on your own, you can just import the default.
+
+```css
+  @import 'ember-right-click-menu';
+  @import 'ember-right-click-menu-default'; /* without any additional styling */
+```
+
+If you don't want to include material icons, add
```
Comment (Haz901) [inline README.md]: `add ....` what?

File: README.md [hunk 2]
```diff
@@ -19,12 +16,43 @@ Installation
...
+
+Preview
+------------------------------------------------------------------------------
+![Ember Right Click Menu](screenshot-1.png)
 
 Usage
 ------------------------------------------------------------------------------
 
-[Longer description of how to use the addon in apps.]
+The right click menu will be applied to it's parent element. Options for the right click menu are added via a simple list of objects that contain a name and an action. You can nest options to create a multi level context menu.
```
Comment (Haz901) [inline README.md]: "~~it's~~ its parent element"

File: addon/services/right-click-menu.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      );
+
+    for (let siblingElement of targetElement.parentElement.children) {
+      this.closePopper(siblingElement);
+    }
+
+    if (
+      targetElement &&
+      popperElement &&
+      // parentElement &&
```
Comment (Haz901) [inline addon/services/right-click-menu.js]: Remove commented code

**Comment by michaelw8** [comment]
[ch328]

**Pull request merged**